.github/CODEOWNERS

```diff
@@ -5,11 +5,9 @@ cpp/               @rapidsai/cuml-cpp-codeowners
 python/            @rapidsai/cuml-python-codeowners
 
 #cmake code owners
-**/CMakeLists.txt          @rapidsai/cuml-cmake-codeowners
-**/cmake/                  @rapidsai/cuml-cmake-codeowners
-python/cuml/pyproject.toml @rapidsai/cuml-cmake-codeowners
-build.sh                   @rapidsai/cuml-cmake-codeowners
-**/build.sh                @rapidsai/cuml-cmake-codeowners
+CMakeLists.txt @rapidsai/cuml-cmake-codeowners
+*.cmake        @rapidsai/cuml-cmake-codeowners
+**/cmake/      @rapidsai/cuml-cmake-codeowners
```

Comment:
updated these rules based on an offline conversation with @vyasr and @dantegd 

`pyproject.toml`-only changes just need `packaging-codeowners` (handled by the `pyproject.toml` rule a few lines down).

This change also adds `cuml-cmake-codeowners:

* when any `.cmake` file is touched
* when any `CMakeLists.txt` file is touched (regardless of depth in the filesystem)

Reply:
Looks good to me.